brain/openfang_adapter.py

```
@@ -540,60 +589,64 @@ async def sync_config(self) -> bool:
                             if resp.status_code == 200:
                                 key_pushed = True
-                                logger.info("[OpenFang] API key synced to openai fallback")
+                                logger.info("[OpenFang] API key synced to %s (format=%s): model=%s",
+                                            provider, type(payload).__name__, model)
                                 break
-                        except Exception:
-                            pass
-
-                if not key_pushed:
-                    logger.warning("[OpenFang] All key push formats failed, relying on config.toml")
+                        except Exception as ep:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)